src/test/java/com/prgrms/nabmart/domain/delivery/DeliveryIntegrationTest.java

```diff
@@ -2,34 +2,92 @@
 import com.prgrms.nabmart.domain.item.Item;
+import com.prgrms.nabmart.domain.item.repository.ItemRepository;
 import com.prgrms.nabmart.domain.item.support.ItemFixture;
 import com.prgrms.nabmart.domain.order.Order;
 import com.prgrms.nabmart.domain.order.OrderItem;
+import com.prgrms.nabmart.domain.order.repository.OrderItemRepository;
+import com.prgrms.nabmart.domain.order.repository.OrderRepository;
 import com.prgrms.nabmart.domain.user.User;
+import com.prgrms.nabmart.domain.user.repository.UserRepository;
 import com.prgrms.nabmart.domain.user.support.UserFixture;
 import java.util.List;
+import java.util.Properties;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.stream.IntStream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 
-public class DeliveryIntegrationTest extends IntegrationTest {
+@SpringBootTest
+public class DeliveryIntegrationTest {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    MainCategoryRepository mainCategoryRepository;
+
+    @Autowired
+    SubCategoryRepository subCategoryRepository;
+
+    @Autowired
+    ItemRepository itemRepository;
+
+    @Autowired
+    OrderItemRepository orderItemRepository;
+
+    @Autowired
+    OrderRepository orderRepository;
+
+    @Autowired
+    DeliveryRepository deliveryRepository;
 
     @Autowired
     DeliveryService deliveryService;
 
+    @Autowired
+    RiderRepository riderRepository;
+
+    @BeforeAll
+    static void beforeAll() {
+        Properties properties = System.getProperties();
+        properties.setProperty("ISSUER", "issuer");
+        properties.setProperty("CLIENT_SECRET", "clientSecret");
+        properties.setProperty("NAVER_CLIENT_ID", "naverClientId");
+        properties.setProperty("NAVER_CLIENT_SECRET", "naverClientSecret");
+        properties.setProperty("KAKAO_CLIENT_ID", "kakaoClientId");
+        properties.setProperty("KAKAO_CLIENT_SECRET", "kakaoClientSecret");
+        properties.setProperty("REDIRECT_URI",
+            "http://localhost:8080/login/oauth2/code/{registrationId}");
+        properties.setProperty("EXPIRY_SECONDS", "60");
+        properties.setProperty("TOSS_SUCCESS_URL", "tossSuccessUrl");
+        properties.setProperty("TOSS_FAIL_URL", "tossFailUrl");
+        properties.setProperty("TOSS_SECRET_KEY", "tossSecretKey");
+        properties.setProperty("REDIS_HOST", "localhost");
+        properties.setProperty("REDIS_PORT", "6379");
+        properties.setProperty("spring.data.redis.host", "localhost");
+        properties.setProperty("spring.data.redis.port", "6379");
```

Comment:
```suggestion
        properties.setProperty("REDIS_HOST", "localhost");
        properties.setProperty("REDIS_PORT", "6379");
        properties.setProperty("spring.data.redis.host", "localhost");
        properties.setProperty("spring.data.redis.port", "6379");
```
REDIS_HOST, spring.data.redis.host 둘 다 필요한가요??

Reply:
해당 환경변수를 설정해주지 않으니 테스트가 안 돌아가서 추가해주었습니다!